.github/workflows/ci.yml

```diff
@@ -34,7 +35,6 @@ jobs:
             echo "UNSUPPORTED RUNNER: $RUNNER_OS"
             exit 1
           fi
-        shell: bash
 
       # Use Go 1.17
```

Comment:
Should we use a newer Go here too and maybe a more sanctioned way of switching to using it?

Reply:
Will do a follow-up PR.

Reply:
Turns out it was already using 1.20. I was just not diffing to current master.
Using the go-action was interfering with the Go cache. So dropped that plan as well.